packages/app/app/actions/queue.ts

```diff
@@ -55,7 +55,7 @@ const getSelectedStreamProvider = (getState) => {
     }
   } = getState();
 
-  return _.find(streamProviders, { sourceName: selected.streamProviders });
+  return _.find(streamProviders, { sourceName: 'Soundcloud' });
```

Comment:
How is the selected stream provider set?

I've set SoundCloud as the default streamProvider - however the selected stream provider is still always set to youtube.

Reply:
It should be retrieved from the plugin settings - if it doesn't work it should be handled in a separate bug. Shouldn't be hardcoded here.